tests/BuilderUrl/input/index.rst

```diff
@@ -8,4 +8,5 @@ Base URL
 .. toctree::
     :glob:
 
+    index
```

Comment:
Because `*` no longer includes "this document" (which is a bug fix), I added `index` manually because the assertions were written to actually want that.